stopify-continuations/src/callcc/delimitTopLevel.ts

```diff
@@ -23,30 +22,15 @@ function delimitStmt(stmt: t.Statement): t.Statement {
     t.memberExpression(runtime, t.identifier('delimit')), [fun]));
 }
 
-/**
```

Comment:
Can you explain why this is safe to remove?

Reply:
Yeah, what's the deal here? I don't see why this is safe.

Reply:
First, I think this is safe to remove because `declVars` turns all declarations of the form `var x = v0` to `var x; ... x = v0`, which means that there are no `var x = v0` that get handled by this piece of code.

Second, I need to remove the code because the assumption that each declaration only declares a single variable is not true when all the variables get declared in one `VariableDeclaration` at the top of the function.